drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -178,51 +187,79 @@ function DragNDropTemplates(url_name) {
-                            ])
-                        ]),
-                        h('div.modal-actions', [
-                            h('button.modal-dismiss-button', gettext("OK"))
+            h('div.keyboard-help-dialog', [
+                h('div.modal-window-overlay'),
+                h('div.modal-window', { attributes: { role: 'dialog', 'aria-labelledby': 'modal-window-title' }, style: {} }, [
+                    h('div.modal-header', [
+                        h('h2.modal-window-title', gettext('Keyboard Help'))
+                    ]),
+                    h('div.modal-content', [
+                        h('p', gettext('You can complete this problem using only your keyboard.')),
+                        h('ul', [
+                            h('li', gettext('Use "Tab" and "Shift-Tab" to navigate between items and zones.')),
+                            h('li', gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.')),
+                            h('li', gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" to drop the item on the current zone.')),
+                            h('li', gettext('Press "Esc" if you want to cancel the drop operation (for example, to select a different item).')),
                         ])
+                    ]),
+                    h('div.modal-actions', [
+                        h('button.modal-dismiss-button', gettext("OK"))
                     ])
                 ])
             ])
         );
     };
 
+    var submitAnswerTemplate = function(ctx) {
+        var attemptsUsedDisplay = ctx.max_attempts > 0 ? 'inline': 'none';
+        var button_enabled = ctx.items.some(function(item) { return item.is_placed}) &&
+            (ctx.max_attempts == 0 || ctx.max_attempts > ctx.num_attempts);
+
+        return (
+          h("section.submit-answer", {}, [
+              h("button.btn-brand.submit-answer-button", { disabled: !button_enabled}, gettext("Submit")),
+              h(
+                  "span.attempts-used", {'style': {'display': attemptsUsedDisplay}},
+                  gettext("You have used {used} of {total} attempts.")
+                      .replace("{used}", ctx.num_attempts).replace("{total}", ctx.max_attempts)
+              )
+          ])
+        );
+    };
+
+    var sidebarTemplate = function(ctx) {
+        return(
+            h("section.sidebar-buttons", {}, [
+                h(
+                    'button.unbutton.btn-brand.btn-small.keyboard-help-button.link-button', { attributes: { tabindex: 0 } },
+                    [
+                        h("i.btn-icon.fa.fa-question"),
+                        gettext('Keyboard Help')
+                    ]
+                ),
+                h(
+                    'button.unbutton.btn-brand..btn-small.reset-button.link-button',
+                    { disabled: ctx.disable_reset_button, attributes: { tabindex: 0 }, 'aria-live': 'off'},
```

Comment:
`aria-live` should be inside `attributes`. Maybe we don't need it at all since this is not a live region anymore?